src/kayenta/report/detail/graph/chartjs/graph.tsx

```diff
@@ -1,17 +1,34 @@
 import * as React from 'react';
-import { Chart } from 'chart.js';
+import { Chart, ChartLegendItem } from 'chart.js';
 
 import { IMetricSetPairGraphProps } from '../metricSetPairGraph.service';
 import { buildChartConfig } from './chartConfigFactory';
 
+export interface ChartLegendItemCallback {
+    (e: MouseEvent, legendItem: ChartLegendItem): void;
+};
+
 export default class ChartJSGraph extends React.Component<IMetricSetPairGraphProps> {
   private canvas: HTMLCanvasElement;
   public chart: Chart;
 
+  public legendOnClickHideAxis : ChartLegendItemCallback = (e: MouseEvent, legendItem: ChartLegendItem): void => {
+    var xAxes = this.chart.config.options.scales.xAxes;
+
+    xAxes.forEach((axis) => {
+      if(axis.scaleLabel.labelString == legendItem.text) {
+        axis.display = ! axis.display;
```

Comment:
```suggestion
        axis.display = !axis.display;
```

Reply:
this would've [for sure gotten fixed by prettier](https://github.com/spinnaker/deck-kayenta/pull/422#pullrequestreview-228100720) if it had run